modules/ui/bin/jenkins-build-deploy.sh

```diff
@@ -36,10 +38,10 @@ exitOnError() {
 }
 
 fromPom() {
-  mvn -f ./modules/$1/pom.xml -P $2 help:evaluate -Dexpression=$3 | sed -n -e '/^\[.*\]/ !{ p; }'
+  mvn -f ./pom.xml help:evaluate -Dexpression=$1 | sed -n -e '/^\[.*\]/ !{ p; }' | tail -n 1
 }
 
-version=`fromPom main ${profile} project.version`
+version=`fromPom project.version`
```

Comment:
@abhishekjain88 This got removed as I was trying different things to make fromPom work to retrieve project.version - I am going to revert all the changes and just keep the change which made retrieving project.version work and which is adding 'tail -n 1'